langchain/llms/huggingface_pipeline.py

```diff
@@ -0,0 +1,101 @@
+"""Wrapper around HuggingFace APIs."""
```

Comment:
ubernit: this looks copied/same as huggingface hub, can we maybe say "huggingface pipelines" or something?